sdk/all/src/main/java/io/opentelemetry/sdk/OpenTelemetrySdk.java

```diff
@@ -158,24 +131,14 @@ public Builder setTracerProvider(TracerProvider tracerProvider) {
      */
     @Override
     public Builder setMeterProvider(MeterProvider meterProvider) {
-      requireNonNull(meterProvider, "meterProvider");
-      this.meterProvider = meterProvider;
+      super.setMeterProvider(meterProvider);
```

Comment:
Should we go ahead and be consistent and only accept `MeterSdkProvider` here since our goal is for this class to only configure the SDK?

Reply:
Yeah, that's probably a good idea. Better to enforce it now even if we don't need to, rather than have to break things later when we do.